examples/ocean_wind_mixing_and_convection.jl

```diff
@@ -109,75 +132,133 @@ nothing # hide
+end
 
-wmax = FieldMaximum(abs, model.velocities.w)
-nothing # hide
+""" Returns colorbar levels equispaced between `clims` and encompassing the extrema of `c`."""
+function sequential_levels(c, clims, nlevels=20)
+    levels = range(clims[1], stop=clims[2], length=nlevels)
+    cmin, cmax = minimum(c), maximum(c)
+    cmin < clims[1] && (levels = vcat([cmin], levels))
+    cmax > clims[2] && (levels = vcat(levels, [cmax]))
+    return clims, levels
+end
 
-# Finally, we set up and run the the simulation.
+# We start the animation at `t = 5minute` since things are pretty boring till then:
 
-simulation = Simulation(model, Δt=wizard, stop_iteration=0, iteration_interval=10)
-simulation.output_writers[:fields] = field_writer
+times = [file["timeseries/t/$iter"] for iter in iterations]
+intro = searchsortedfirst(times, 5minute)
 
-anim = @animate for i in 1:100
-    ## Run the simulation forward
-    simulation.stop_iteration += 10
-    walltime = @elapsed run!(simulation)
+anim = @animate for (i, iter) in enumerate(iterations[intro:end])
 
-    ## Print a progress message
-    @printf("i: %04d, t: %s, Δt: %s, wmax = %.1e ms⁻¹, wall time: %s\n",
-            model.clock.iteration, prettytime(model.clock.time), prettytime(wizard.Δt),
-            wmax(model), prettytime(walltime))
+    @info "Drawing frame $i from iteration $iter..."
 
-    ## Coordinate arrays for plotting
-    xC, zF, zC = xnodes(Cell, grid)[:], znodes(Face, grid)[:], znodes(Cell, grid)[:]
+    t = file["timeseries/t/$iter"]
+    w = file["timeseries/w/$iter"][:, 1, :]
+    T = file["timeseries/T/$iter"][:, 1, :]
+    S = file["timeseries/S/$iter"][:, 1, :]
+    νₑ = file["timeseries/νₑ/$iter"][:, 1, :]
 
-    ## Slices to plots.
-    jhalf = ceil(Int, model.grid.Ny/2)
-    w = Array(interior(model.velocities.w))[:, jhalf, :]
-    T = Array(interior(model.tracers.T))[:, jhalf, :]
-    S = Array(interior(model.tracers.S))[:, jhalf, :]
+    wlims, wlevels = divergent_levels(w, 1e-1)
+    Tlims, Tlevels = sequential_levels(T, (19.7, 20.0))
+    Slims, Slevels = sequential_levels(S, (34.9999, 35.002))
+    νlims, νlevels = sequential_levels(νₑ, (1e-9, 1e-2))
 
-    ## Plot the slices.
-    w_plot = heatmap(xC, zF, w', xlabel="x (m)", ylabel="z (m)", color=:balance, clims=(-3e-2, 3e-2))
-    T_plot = heatmap(xC, zC, T', xlabel="x (m)", ylabel="z (m)", color=:thermal, clims=(19.75, 20.0))
-    S_plot = heatmap(xC, zC, S', xlabel="x (m)", ylabel="z (m)", color=:haline,  clims=(34.99, 35.01))
+    kwargs = (linewidth=0, xlabel="x (m)", ylabel="z (m)", aspectratio=1,
+              xlims=(0, grid.Lx), ylims=(-grid.Lz, 0))
 
+    w_plot = contourf(xw, zw, w'; color=:balance, clims=wlims, levels=wlevels, kwargs...)
+    T_plot = contourf(xT, zT, T'; color=:thermal, clims=Tlims, levels=Tlevels, kwargs...)
+    S_plot = contourf(xT, zT, S'; color=:haline,  clims=Slims, levels=Slevels, kwargs...)
+    ν_plot = heatmap(xT, zT, νₑ'; color=:thermal, clims=νlims, levels=νlevels, kwargs...)
```

Comment:
```suggestion
    ν_plot = contourf(xT, zT, νₑ'; color=:thermal, clims=νlims, levels=νlevels, kwargs...)
```

Would this look better with `contourf`?

![image](https://user-images.githubusercontent.com/20099589/97160601-f7f94e00-1752-11eb-847e-b65449026da4.png)

Reply:
Ah just saw your comment about `contourf` being slower :(

Reply:
Ah, there's another reason to use heatmap here --- I think it's actually useful to see the grid point values of the eddy viscosity. I'll add a comment about this in the example.